tensorflow/python/keras/metrics.py

```diff
@@ -2272,6 +2272,10 @@ def update_state(self, y_true, y_pred, sample_weight=None):
     Returns:
       Update op.
     """
+    # Converted y_true and y_pred into Tensors
+    y_true = ops.convert_to_tensor(y_true)
+    y_pred = ops.convert_to_tensor(y_pred)
+    
```

Comment:
Can we add a test case for list inputs? Sorry, i forgot to mention this in my first review.

Reply:
Sure. You mean checking for list inputs, right?